### ISSUE TITLE
 No notifications, activities and no mails in collective app generated after update to NC 25 (other notifications and mails working) 

### ISSUE THREAD

**Issue by sonic42** [nextcloud/collectives#641]
Hi,

with NC 24 activity’s and notification mails was generated if a collective is created, the user is added to the new collective and the content in a collective was changed. After the update, no activity events in activity app and no more mails from collective app are generated. All other notifications and mails (file created, file shared and calendar events) are working.

**To Reproduce**
1. Go to colletive app 
2. Create new colletive
3. Add users to collective
4. no events an mails generated
(all notifications are enabled in configuration. User and system) 

This behavior is reproducible in other updated instances. 

**Expected behavior**
Activity’s and notification mails generated to inform users. 

NC 25.0.5
Activity 2.17.0
Collectives 2.4.0
PHP 8.1.16
Apache 2.4.57

**Comment by EWPrivat** [comment]
Same issue here.
NC 30.0.4
Collectives 2.15.2

**Comment by joshtrichards** [comment]
See #317 
And #2541

**Comment by mejo-** [comment]
Indeed, let's close this as duplicate, also of https://github.com/nextcloud/collectives/issues/1185 (Activity app support).
